simulators/active_surface.py

```diff
@@ -175,18 +178,17 @@ def __init__(self, driver_reset_delay=0):
         self.drivers = [Driver(driver_reset_delay) for x in xrange(32)]
 
     def create_empty_msg(self):
-        #parsing operation variables
         self.msg = b''
         self.msg_to_all = False
         self.expected_bytes = 0
 
     def parse(self, byte):
-        #Return None or the response.
-        #Raise a ValueError in case of unexpected data
+        # Return None or the response.
```

Comment:
Mmm, this comment is out of date.  This method returns `True`, `False` or raises `ValueError`.  You should remove the comment and write instead a docstring or just point out to the [issue](https://github.com/discos/simulators/issues/1).  I am setting up the test and documentation environment.  The doc environment will read the docstring from the code and generate the API documentation. So if you write the docstring we will have:

* the code documented
* the HTML documentation already done
* the internal report (it gives you some points for your future concourses...) almost done, because we can just generate it from the doc